### PR TITLE
Support atom tenant types

### PR DESCRIPTION
**Pull request by alextargov**
**Description**

Changes proposed in this pull request:
- Add support for 3 new tenants

**Related issue(s)**

- ...

**Pull Request status**

- [ ] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/pkg/tenant/entity.go [hunk 1]
```diff
@@ -28,6 +28,12 @@ const (
 	Account Type = "account"
 	// Subaccount tenants must have a parent of type Account.
 	Subaccount Type = "subaccount"
+	// Organization tenants can be parents of Folder tenants.
```
Comment (kaloyanspiridonov) [inline components/director/pkg/tenant/entity.go:31]: // Organization tenants can be parents of Folder tenants. ->  // Organization tenants can be parents of Folder or ResourceGroup tenants.

File: components/director/pkg/tenant/entity.go [hunk 2]
```diff
@@ -28,6 +28,12 @@ const (
...
+	Organization Type = "organization"
+	// Folder tenants must have a parent of type Organization.
+	Folder Type = "folder"
+	// ResourceGroup tenants must have a parent of type Folder.
```
Comment (kaloyanspiridonov) [inline components/director/pkg/tenant/entity.go:35]: // ResourceGroup tenants must have a parent of type Folder. -> // ResourceGroup tenants must have a parent of type Folder or organization.

**Review by kaloyanspiridonov — CHANGES_REQUESTED** [review]

**Review by la4ezar — APPROVED** [review]

**Review by kaloyanspiridonov — APPROVED** [review]

**Pull request merged**